data/COPYRIGHTS.md

```diff
@@ -0,0 +1,34 @@
+## Copyrights and Attribution
+
+### Ohsome Copyrights and Attribution - OpenStreetMap
+
+https://ohsome.org/copyrights
```

Comment:
Should the content of https://ohsome.org/copyrights be copied over to this docyment or is a link enough for now?

Reply:
Something like that?
```suggestion
### Aggregated OpenStreetMap Data: ohsome API

- Source: https://api.ohsome.org
- Attribution and use conditions: https://ohsome.org/copyrights
- Citation: https://github.com/GIScience/ohsome-api#how-to-cite
```